src/satellites.jl

```diff
@@ -231,6 +231,48 @@ function getTotalSize(loc, md, printLevel=0)
     return totalSize, inSize, runSum
 end
 
+
+function getTotalSize(loc, fn::String, printLevel=0)
```

Comment:
can you add a documentation here?

Reply:
is there any way that we could combine both overloaded `getTotalSize` functions?

Reply:
yes, now combined